Lib/test/support/__init__.py

```diff
@@ -2578,30 +2578,29 @@ def sleeping_retry(timeout, err_msg=None, /,
         delay = min(delay * 2, max_delay)
 
 
-class CPUStopwatch:
+class Stopwatch:
     """Context manager to roughly time a CPU-bound operation.
 
-    Disables GC. Uses CPU time if it can (i.e. excludes sleeps & time of
-    other processes).
+    Disables GC. Uses perf_counter, i.e. a clock with the highest
+    available resolution. It is chosen even though it does include
+    time elapsed during sleep and is system-wide, because the
+    resolution of process_time is too coarse on Windows and
+    process_time does not exist everywhere (e.g. WASM).
 
     N.B.:
-    - This *includes* time spent in other threads.
     - Some systems only have a coarse resolution; check
-      stopwatch.clock_info.rseolution if.
+      stopwatch.clock_info.resolution if.
```

Comment:
Oops, the comment was incomplete
```suggestion
      stopwatch.clock_info.resolution when using the results.
```

Reply:
Yeah, let's fix it when we are here. No need to keep the diff small ...